blogs/scripts/taxonomy.js

```diff
@@ -0,0 +1,29 @@
+import { toClassName } from './lib-franklin.js';
+
+async function getTags(ul) {
```

Comment:
I think a simple array of strings is not going to be sufficient for future state. I think each tag needs to be an object with at minimum and id and a title. in the future, we can enhance this structure to include additional attributes (Looking at https://www.keysight.com/clientapi/search/aemtags/en I can forsee eventually needing language code, and possibly path as well)

Reply:
Since the initial focus was on tags.docx, which is just HTML lists of strings, this solution fits.

The AEM tags come in as JSON with additional properties per tag, which is more in line with your thoughts. This file will be changing to reflex that in the next iteration.